app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -257,21 +235,30 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
      */
     fun performSearch(query: String): Boolean {
 
-        for ((pos, item) in skills.withIndex()) {
-            if (query in item.first) {
-                skillMetrics.scrollToPosition(pos)
-                return true
-            }
+        var searchedSkillsList: ArrayList<SkillData> = arrayListOf()
 
-            for (item2 in item.second) {
-                if (query.toLowerCase() in item2.group.toLowerCase()) {
-                    skillMetrics.scrollToPosition(pos)
-                    return true
+        for ( skill_ in skills) {
```

Comment:
We don't use "skill_" this kind of variable name. It should be "skill"

Reply:
Yes, I know. I was testing something so I used that, I'll change that.